Gemfile

```diff
@@ -47,8 +44,6 @@ group :development do
   gem 'rake', '~> 10.0.4'
   gem 'pry', require: false
   gem 'pry-byebug', require: false
-  #gem 'pry-stack_explorer', require: false
-  #gem 'pry-exception_explorer', require: false
```

Comment:
No, not really.

On Wednesday, August 5, 2015, Paul Robinson <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/redhat-developer/developers.redhat.com/pull/348#discussion_r36330854>
> :
>
> > @@ -47,8 +44,6 @@ group :development do
> >    gem 'rake', '~> 10.0.4'
> >    gem 'pry', require: false
> >    gem 'pry-byebug', require: false
> > -  #gem 'pry-stack_explorer', require: false
> > -  #gem 'pry-exception_explorer', require: false
>
> @LightGuard <https://github.com/LightGuard> Useful during debugging?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/348/files#r36330854>
> .
>


-- 
Sent from Gmail Mobile